staging/src/k8s.io/api/networking/v1/well_known_annotations.go

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+const (
+	// AnnotationIsDefaultIngressClass can be used to indicate that an
+	// IngressClass should be considered default. When a single IngressClass
+	// resource has this annotation set to true, new Ingress resources without a
+	// class specified will be assigned this default class.
+	AnnotationIsDefaultIngressClass = "ingressclass.kubernetes.io/is-default-class"
+
+	// AnnotationIngressClass indicates the class of an Ingress to be used when
+	// determining which controller should implement the Ingress. Use of this
+	// annotation is deprecated. The Ingress class field should be used instead
+	// of this annotation.
+	// +deprecated
+	AnnotationIngressClass = "kubernetes.io/ingress.class"
```

Comment:
this was intentionally omitted from v1 because there is now a field for this value

Reply:
removed

Reply:
do we need the admission controller now?

Reply:
it seems that the admission is needed for the default-class

Reply:
@liggitt @bowei @robscott do I need to update the API docs to remote the annotation precedence over the field?

https://github.com/kubernetes/kubernetes/blob/5c586201186b1f31d16257511baef7793112ae76/pkg/apis/networking/types.go#L249-L260